packaging_automation/tests/test_citus_package.py

```diff
@@ -39,7 +40,7 @@
 def setup_module():
     if not os.path.exists(PACKAGING_SOURCE_FOLDER):
         run(
-            f"git clone --branch all-citus https://github.com/citusdata/packaging.git {PACKAGING_SOURCE_FOLDER}")
+            f"git clone --branch all-citus-unit-tests https://github.com/citusdata/packaging.git {PACKAGING_SOURCE_FOLDER}")
```

Comment:
maybe add comment like that:
```python
# Run tests against "all-citus-unit-tests" since we don't want to deal with the changes
# made to "all-citus" in each release.
```

Reply:
Fixed